### PR TITLE
feat(evmutil)!: implement MsgConvertCosmosCoinToERC20

### PR DESCRIPTION
**Pull request by pirtleshell**
## Description
* adds message server handler for MsgConvertCosmosCoinToERC20
* registers the message for codec
* includes CLI cmd
* intend follow-up prs for the following:
  * event emission
  * e2e test

```
kava tx evmutil convert-cosmos-coin-to-erc20 --help
converts asset native to Cosmos Co-chain to an ERC20 on the EVM Co-chain

Usage:
  kava tx evmutil convert-cosmos-coin-to-erc20 [receiver_0x_address] [amount] [flags]

Examples:
Convert 500 ATOM and send ERC20 to 0x03db6b11F47d074a532b9eb8a98aB7AdA5845087:
  kava tx evmutil convert-cosmos-coin-to-erc20 0x03db6b11F47d074a532b9eb8a98aB7AdA5845087 500000000ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2 --from <key> --gas 2000000
```

Checked out some initial gas things:
* 1st convert with deploy used 1,100,009 gas
* 2nd convert (which just transfers sdk.Coin & mints ERC20) used 152,648 gas

## Checklist
 - [x] Changelog has been updated as necessary.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -40,6 +40,10 @@ Ref: https://keepachangelog.com/en/1.0.0/
 - (evmutil) [#1590] & [#1596] Add allow list param of sdk native denoms that can be transferred to evm
 - (evmutil) [#1591] & [#1596] Configure module to support deploying ERC20KavaWrappedCosmosCoin contracts
 - (evmutil) [#1598] Track deployed ERC20 contract addresses for representing cosmos coins in module state
+- (evmutil) [#1603] Add MsgConvertCosmosCoinToERC20 for converting representing an sdk.Coin as an ERC20 in the EVM
```
Comment (galxy25) [inline CHANGELOG.md:43]: `for converting`

Comment (galxy25) [reply CHANGELOG.md:43]: or `for representing` 😅 not sure

File: CHANGELOG.md [hunk 2]
```diff
@@ -40,6 +40,10 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
+
+### Client Breaking
+- (evmutil) [#1603] Renamed error `ErrConversionNotEnabled` to `ErrEVMConversionNotEnabled`
```
Comment (galxy25) [inline CHANGELOG.md:46]: 😃 big fan of explicitness as a pillar of reliable distributed systems design (read: all web and blockchain software)

File: x/evmutil/client/cli/tx.go [hunk 3]
```diff
@@ -121,3 +122,40 @@ func getCmdConvertERC20ToCoin() *cobra.Command {
...
 	}
 }
+
+func getCmdMsgConvertCosmosCoinToERC20() *cobra.Command {
+	return &cobra.Command{
+		Use:   "convert-cosmos-coin-to-erc20 [receiver_0x_address] [amount] [flags]",
+		Short: "converts asset native to Cosmos Co-chain to an ERC20 on the EVM Co-chain",
+		Example: fmt.Sprintf(
+			`Convert 500 ATOM and send ERC20 to 0x03db6b11F47d074a532b9eb8a98aB7AdA5845087:
+  %s tx~ %s convert-cosmos-coin-to-erc20 0x03db6b11F47d074a532b9eb8a98aB7AdA5845087 500000000ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2 --from <key> --gas 2000000`,
```
Comment (rhuairahrighairidh) [inline x/evmutil/client/cli/tx.go:132]:
```suggestion
  %s tx %s convert-cosmos-coin-to-erc20 0x03db6b11F47d074a532b9eb8a98aB7AdA5845087 500000000ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2 --from <key> --gas 2000000`,
```
nit: typo?

**Review by rhuairahrighairidh — APPROVED** [review]

**Pull request merged**